util/gnmi.go

```diff
@@ -196,3 +196,27 @@ func FindModelData(mods []*yang.Entry) ([]*gpb.ModelData, error) {
 
 	return modelData, nil
 }
+
+// JoinPaths joins an prefix and suffix paths, returning an error if their
+// target or origin fields don't match.
```

Comment:
... returning an error if their
// target or origin fields **are both non-empty and** don't match.

Reply:
Done.